apps/web/package.json

```diff
@@ -130,7 +130,8 @@
     "uniqid": "^5.3.0",
     "uuid": "8.3.2",
     "web-vitals": "^1.0.1",
-    "zod": "^3.22.4"
+    "zod": "^3.22.4",
+    "@novu/js": "workspace:*"
```

Comment:
Let's try it in a playground app, not in web until we release.

Reply:
Didn't mean to commit that